plugins/rich-editor/src/scripts/components/ParagraphEditorToolbar.jsx

```diff
@@ -73,7 +75,11 @@ export class ParagraphEditorToolbar extends React.PureComponent {
 
         // Quill can directly on the class as it won't ever change in a single instance.
         this.quill = props.quill;
-
+        UniqueID.enableUniqueIds(this);
```

Comment:
The withEditor function injects the editors common unique id in as a prop. You should be able to get with `this.props.editorID`. Take a look at the EmojiPopover for an example.